config/environments/production.rb

```diff
@@ -28,7 +28,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
This one is kind of weird to me.
As I checked, it seems that you're precompiling the asset already 🤔

Ref: https://stackoverflow.com/questions/51944864/config-assets-compile-false-makes-heroku-not-display-my-images-in-production

Reply:
@Lahphim Well the reason as to why I did this is because I couldn't get the styles to load up on the site 😖, I got the idea from the previous IC's 
- [Su.v](https://github.com/nimblehq/internal-certification-ruby/blob/edcb4ce08625d899860ba56f4bd8a3285bcb9176/su.v/config/environments/production.rb#L31)

<img width="545" alt="Screen Shot 2022-09-19 at 2 54 51 PM" src="https://user-images.githubusercontent.com/28055566/190973063-e84c0cd5-ad72-4ed5-b089-e1f39709a8c2.png">

This is what shows when I try to access the CSS file on Heroku, the site itself loads up but the styles are in fact missing. The fix I found for it was to change `config.assets.compile` to `true`, that was the only fix I could find for it 😬 

I tried the [fix in the reference](https://stackoverflow.com/questions/51944864/config-assets-compile-false-makes-heroku-not-display-my-images-in-production) you've linked but still no luck x.x The site is still rendering a page without being able to find the CSS file 😖 

<img width="325" alt="Screen Shot 2022-09-19 at 2 57 45 PM" src="https://user-images.githubusercontent.com/28055566/190973640-5e4f4065-0958-421e-8427-fad6a4392ecd.png">